contrib/test/run_test_vectors.sh

```diff
@@ -33,6 +33,9 @@ cat contrib/test/syscall-fixtures/*.list | xargs -n 1000 ./$OBJDIR/unit-test/tes
 LOG=$LOG_PATH/test_exec_precompiles
 cat contrib/test/precompile-fixtures/*.list | xargs -n 1000 ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG
 
+LOG=$LOG_PATH/test_exec_precompiles
```

Comment:
@mjain-jump too late sorry, but for the next one this should be test_exec_txn :)